contracts/seed/Seed.sol

```diff
@@ -155,20 +155,26 @@ contract Seed {
 
     /**
       * @dev                     Buy seed tokens.
-      * @param _amount           The amount of tokens to buy.
+      * @param _seedAmount       The amount of seed tokens to buy.
     */
-    function buy(uint256 _amount) public protected checked {
-        require((fundingToken.balanceOf(address(this)).add(_amount)) <= cap, "Seed: amount exceeds contract sale cap");
-        require(fundingToken.transferFrom(msg.sender, address(this), _amount), "Seed: no tokens");
+    function buy(uint256 _seedAmount) public protected checked {
+        //        TODO: DEAR ZNATOKI, Attention, question from our viewers, why do we need to divide anything anyway
+        uint256 fundingAmount = (_seedAmount.mul(price)).div(PCT_BASE);
+        require((fundingToken.balanceOf(address(this)).add(fundingAmount)) <= cap,
```

Comment:
please include fee to the check

Reply:
Included, but actually have one more question, I think we do need to include price of the fee tokens to overall price in the fundingTokens, that we are sending to the contract?

Reply:
And why are we paying fee in SeedTokens anyways and not in the FundingTokens?